specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/databaseVulnerabilityAssessments.json

```diff
@@ -236,7 +236,7 @@
           ]
         },
         "storageAccountAccessKey": {
-          "description": "Specifies the identifier key of the auditing storage account. If 'StorageContainerSasKey' isn't specified, storageAccountAccessKey is required.",
+          "description": "Specifies the identifier key of the vulnerability assessment storage account. If 'StorageContainerSasKey' isn't specified, storageAccountAccessKey is required.",
```

Comment:
1. should be updated in the code too
2. sync with Tal to update the server level swagger